index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <title>SA AUNG HTET NYEIN</title>
+  </head>
+  <body>
+    <!-- the toolbar -->
+    <nav>
+      <p class="logo">SASA</p>
+      <div class="menu"><i class="fa-solid fa-bars"></i></div>
+    </nav>
+
+    <!-- headline -->
+    <section id="headline">
+      <h1 class="big-intro">Hey There. I’m SASA</h1>
+      <h5 class="small-intro">I’am a Software Developer</h5>
+      <p class="about-me">
+        A software engineering student with holistic knowledge of software
+        development with over 4 years of web development experience. Passionate
+        about UI/UX design and the art of photography. Extremely motivated to
+        develop my skills and grow professionally.
+      </p>
+      <div class="social-accounts">
+        <a href="#">
+          <i class="fa-brands fa-github"></i>
+        </a>
```

Comment:
- [ ] Great job wrapping your icons in an `a` tag. 👏  Kindly wrap the icons in a `ul` tag and each of the icons in an `li` tag 🙏